### DISCUSSION TITLE
Does ccache support Qt meta-object compiler (aka moc)?

### DISCUSSION THREAD

**Discussion by kirmitrahovich** [ccache/ccache#1062] [Questions & answers]
Hello, ccache community!

A bit of info about Qt moc: https://doc.qt.io/qt-5/moc.html

Is anyone working on it or has been tried? How hard it might be? 

Best regards
Kirill

**Comment by wbickford-efj** [comment]
+1: this would be useful. Although I wonder if moc.exe falls outside the scope of a "C/C++ compiler cache".

Note: With Qt 6, they started adopting cmake. https://cmake.org/cmake/help/latest/prop_tgt/AUTOMOC.html

**Comment by cristianadam** [comment]
ccache doesn't support moc at the moment. 

This is similar to the yacc/bison issue at https://github.com/ccache/ccache/pull/1156

At some point in the future this will be possible with the compiler abstraction from https://github.com/ccache/ccache/issues/956
